app/views/users/show.html.erb

```diff
@@ -1,2 +1,23 @@
-<h1><%= @user.name %></h1>
+<div class="card mb-3">
+  <div class="d-flex">
+    <img src="<%= @user.photo %>" alt="avatar" />
+    <div class="card-body">
+      <div class="card-title">
+        <h3 class="m-0"><b><%= @user.name %></b></h3>
+      </div>
+      <div class="d-flex h-50 justify-content-end align-items-end">
+        <p class="m-0">Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+  </div>
+</div>
 
+<div class="recent-posts">
+  <h4>Recent Posts</h4>
+  <ul>
+  <% @posts.each do |post| %>
+    <li><%= link_to post.title, user_post_path(@user, post) %></li>
+  <% end %>
+</ul>
+
```

Comment:
- [ ] Please make sure on the user show page to add a link to see `See all Post` now I am not able to see all posts

Reply:
- [ ] [REQUIRED] This has been resolved, however, we need to ensure that our design sticks to the wireframe provided.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="367" alt="Screenshot 2023-08-24 at 19 32 24" src="https://github.com/ShaliniNayan/Blog-app/assets/58692590/3b0781f7-4e11-4f37-9e33-e7ffce2b0803"> | <img width="466" alt="Screenshot 2023-08-24 at 19 56 36" src="https://github.com/ShaliniNayan/Blog-app/assets/58692590/8459f546-f0a0-4e04-b16d-18327f80064e">